v2.8.0/language-es.yml

```diff
@@ -0,0 +1,300 @@
+# Language: es
+
+msg-shop-type-admin-regular: "Admin shop"
+msg-shop-type-selling: "Selling shop"
+msg-shop-type-buying: "Buying shop"
+msg-shop-type-trading: "Trading shop"
+msg-shop-type-book: "Book shop"
+
+msg-shop-type-desc-admin-regular: "tiene stock ilimitado"
+msg-shop-type-desc-selling: "vende items a los jugadores"
+msg-shop-type-desc-buying: "compra items de los jugadores"
+msg-shop-type-desc-trading: "tradea items con jugadores"
+msg-shop-type-desc-book: "vende copias de libro"
+
+msg-shop-object-type-living: "{type}"
+msg-shop-object-type-sign: "sign"
+msg-shop-object-type-npc: "npc"
+
+msg-selected-shop-type: "&aTipo de tienda seleccionada: &6{type} &7({description})"
+msg-selected-shop-object-type: "&aTipo de objeto seleccionado: &6{type}"
+
+msg-creation-item-selected: |
+    &aCreación de Tienda:
+    &e  Click izquierdo/derecho para seleccionar el tipo de tienda.
+    &e  Shift + click izquierdo/derecho para seleccionar el tipo de objeto.
+    &e  Click derecho en un cofre para seleccionarlo. 
+    &e  Click derecho en un bloque para spawnear al aldeano.
+msg-button-previous-page: "&6<- Página anterior ({prev_page} de {max_page})"
+msg-button-previous-page-lore: []
+msg-button-next-page: "&6Página siguiente ({next_page} de {max_page}) ->"
+msg-button-next-page-lore: []
+msg-button-current-page: "&6Página {page} de {max_page}"
+msg-button-current-page-lore: []
+
+msg-button-name: "&aEstablacer el nombre de la tienda"
+msg-button-name-lore:
+- Reenombra a
+- tu tienda
+msg-button-chest: "&aVer inventario del cofre"
+msg-button-chest-lore:
+- Lets you view the inventory
+- your shopkeeper is using
+msg-button-delete: "&4Eliminar"
+msg-button-delete-lore:
+- Cierra y elimina
+- esta tienda
+msg-button-hire: "&aContratar"
+msg-button-hire-lore:
+- Compra esta tienda
+
+msg-button-sign-variant: "&aElegir versión del cartel"
+msg-button-sign-variant-lore:
+- "Cambia el tipo de"
+- "madera de los carteles"
+msg-button-baby: "&aAlterna entre aldeano bebé/adulto"
```

Comment:
"aldeano": This is not only used for villagers, but all types of mobs that support baby variants